### ISSUE TITLE
Connection error

### ISSUE THREAD

**Issue by amankharb** [tgalal/yowsup#1369]
File "build/bdist.linux-x86_64/egg/yowsup/common/http/warequest.py", line 73, in send
  File "build/bdist.linux-x86_64/egg/yowsup/common/http/warequest.py", line 111, in sendGetRequest
  File "build/bdist.linux-x86_64/egg/yowsup/common/http/warequest.py", line 167, in sendRequest
  File "/usr/lib/python2.7/httplib.py", line 1048, in request
    self._send_request(method, url, body, headers)
  File "/usr/lib/python2.7/httplib.py", line 1088, in _send_request
    self.endheaders(body)
  File "/usr/lib/python2.7/httplib.py", line 1044, in endheaders
    self._send_output(message_body)
  File "/usr/lib/python2.7/httplib.py", line 888, in _send_output
    self.send(msg)
  File "/usr/lib/python2.7/httplib.py", line 850, in send
    self.connect()
  File "/usr/lib/python2.7/httplib.py", line 1261, in connect
    HTTPConnection.connect(self)
  File "/usr/lib/python2.7/httplib.py", line 831, in connect
    self.timeout, self.source_address)
  File "/usr/lib/python2.7/socket.py", line 575, in create_connection
    raise err
socket.error: [Errno 111] Connection refused

**Comment by jlguardi** [comment]
It seems that you have a problem with your internet access. Probably  your trafic is filtered?
Try from python console:
```
>>> import urllib2
>>> url='http://hb.whatsapp.net/hb.txt'
>>> urllib2.urlopen(url).read()
'a\n'
```

**Comment by amankharb** [comment]
Yes I am behind my college proxy. I have set env var by export http. Still the same error. Please help me. I have an assignment to do by today.

**Comment by amankharb** [comment]
I did this. Got same output. How does this helps. I am getting same error.
